ThreeBotPackages/threefold/directory/actors/nodes.py

```diff
@@ -61,16 +78,59 @@ def list(self, farm, country, city, cru, sru, mru, hru, schema_out):
     def get(self, node_id, schema_out):
         """
         ```in
-        node_id = (S)
+        node_id = (I)
```

Comment:
This is not valid. The node_id type is a string.
check the model: https://github.com/threefoldtech/jumpscaleX_threebot/blob/52ee31524616f411c4f572dcb52a5b6abafd6969/ThreeBotPackages/threefold/directory/models/tfgrid_node_iface_1.toml#L30

If you want more detail about 0-OS identity you can also check the documentation: https://github.com/threefoldtech/zosv2/blob/master/docs/identity/readme.md